.github/workflows/build-and-push-nosql-sec-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and publish NoSQL container image to GitHub Container Registry
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'NoSQL/ce-sec/*'
+      - '.github/workflows/build-and-push-nosql-image.yml'
```

Comment:
This path doesn't match the path of this file.

Reply:
I tested it and working on my fork, so for me the path is good

![image](https://user-images.githubusercontent.com/66302636/186142003-f8208c9a-f105-4723-b545-7b017da7700c.png)

There is probably a misunderstanding here: there is no old workflow and a new one - we want to maintain two versions